package/firmware/rcw/Makefile

```diff
@@ -78,6 +78,7 @@ define Build/Configure
 endef
 
 define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR)
 endef
```

Comment:
you can remove the "define Build/Compile" block completely, this is more or less what "define Build/Compile/Default" does.